Makefile

```diff
@@ -24,10 +24,10 @@ ASSISTED_ORG ?= quay.io/ocpmetal
 ASSISTED_TAG ?= latest
 CONTAINER_BUILD_PARAMS = --network=host --label git_revision=${GIT_REVISION} ${CONTAINER_BUILD_EXTRA_PARAMS}
 
-SERVICE := $(or ${SERVICE},${ASSISTED_ORG}/assisted-service:${ASSISTED_TAG})
+export SERVICE := $(or ${SERVICE},${ASSISTED_ORG}/assisted-service:${ASSISTED_TAG})
```

Comment:
This change should be tested on PSI and deployment from laptop.

Reply:
This change is required for the subsystem to use the SERVICE specified at the jenkins file and not latest.
@YuviGold I think we should just revert the change that was pushed yesterday.